components/integration/development/kustomization.yaml

```diff
@@ -10,6 +10,11 @@ images:
   newName: quay.io/redhat-appstudio/integration-service
   newTag: c6ceab5e54a3b57e1a699d534ad1179ac60029f2
 
+configMapGenerator:
+- name: console-url
+  literals:
+    - CONSOLE_URL="devUrlChangeInFuture"
```

Comment:
I think this might break when getting processed on the [integration-service side](https://github.com/redhat-appstudio/integration-service/pull/609/files#diff-e47e5b0d6143994b257b1b2d54a0525a6470ac5636ffebc2452517a7e367abbdR213).
Can we simply not set this envvar in development instances for starters?